bing_search.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+# !/usr/bin/env ruby
+
+require 'rest-client'
+
+# A simple Http Request using Rest-Client Bing search for ruby
+
+class BingSearch
+  attr_accessor :url, :querry
+
+  def initialize(url, *querry)
+    @url = url
+    @querry = querry || ' '
+  end
```

Comment:
Since you are accepting a variable number of parameters for the init method, why not stick to the same type?

On line 14, @query is either an array or a string. You should remember that when you pass only 1 argument to the init method, that query would be `[]` (an empty array).

So consider sticking to array or strings only. If you use string, then it would be used thus; `BingSearch.new(url, 'query1 query2 query3')`